portal/index.php

```diff
@@ -463,7 +431,7 @@ function validate_new_pass() {
                 <?php $datetimepicker_timepicker = false; ?>
                 <?php $datetimepicker_showseconds = false; ?>
                 <?php $datetimepicker_formatInput = false; ?>
-                <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
+                <?php include $GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'; ?>
```

Comment:
why change this to an include?

Reply:
Agree. I've found the best use of include is if you want to handle problems with include loading in code. Sometimes you don't want your app to stop cold as a require will do but seek other ways to continue app flow. In this case, require is appropriate.

Reply:
What's weird is I didn't actually modify this line - the code linter did this 😂

Reply:
that is weird, pretty sure it shouldn't be doing that
what codesniffer commands did you use?

Reply:
Just edited it back now and it's not spitting out any errors - that was weird, probably executed the command wrong at one point so let me know if there are any other errors before I commit my last commit for this pull request.

Reply:
Nope you're good. Once travis is finish with your commit, i'll merge to master. I am going to be working with appointments next for handling recurring events.